src/BlockID.h

```diff
@@ -459,6 +459,7 @@ enum ENUM_ITEM_ID : short
 	E_ITEM_GOLD_HORSE_ARMOR = 418,
 	E_ITEM_DIAMOND_HORSE_ARMOR = 419,
 	E_ITEM_LEAD = 420,
+	E_ITEM_LEASH = E_ITEM_LEAD,
```

Comment:
Cause he/she said so :smile: 
Not very important think to argue about, others did it before :stuck_out_tongue: 
https://english.stackexchange.com/questions/273617/difference-between-lead-and-leash